docs/zh_cn/1_exist_data_model.md

```diff
@@ -487,17 +487,35 @@ pipelines = [
 
 MMDetection 也为训练检测模型提供了开盖即食的工具。本节将展示在标准数据集（比如 COCO）上如何训练一个预定义的模型。
 
-重要信息：在配置文件中的学习率是在 8 块 GPU，每块 GPU 有 2 张图像（批大小为 8*2=16）的情况下设置的。
-根据 [线性扩展规则](https://arxiv.org/abs/1706.02677) ，如果你使用不同数目的 GPU 或者每块 GPU 上有不同数量的图片，你需要设置学习率以正比于批大小，比如，
-在 4 块 GPU 并且每张 GPU 上有 2 张图片的情况下，设置 `lr=0.01`； 在 16 块 GPU 并且每张 GPU 上有 4 张图片的情况下, 设置 `lr=0.08`。
-
 ### 数据集
 
 训练需要准备好数据集，细节请参考 [数据集准备](#数据集准备) 。
 
 **注意**：
 目前，`configs/cityscapes` 文件夹下的配置文件都是使用 COCO 预训练权值进行初始化的。如果网络连接不可用或者速度很慢，你可以提前下载现存的模型。否则可能在训练的开始会有错误发生。
 
+### 学习率自动缩放
+
+**注意**：在配置文件中的学习率是在 8 块 GPU，每块 GPU 有 2 张图像（批大小为 8*2=16）的情况下设置的。其已经设置在`config/_base_/default_runtime.py` 中的 `default_batch_size`。当配置文件的批次大小为`16`时，学习率自动缩放基于该值。同时，为了不影响其他基于 mmdet 的 codebase，启用自动缩放标志 `enable_auto_scale_lr` 默认设置为 `False`。
+
+如果要启用此功能，需在命令添加参数 `--auto-scale-lr`。并且在启动命令之前，请检查下即将使用的配置文件的名称，因为配置名称指示默认的批处理大小。
+在默认情况下，批次大小是 `8 x 2 = 16`，例如：`faster_rcnn_r50_caffe_fpn_90k_coco.py` 或者 `pisa_faster_rcnn_x101_32x4d_fpn_1x_coco.py`；若不是默认批次，你可以看到像 `_NxN_` 字样的，例如：`faster_rcnn_x101_32x4d_fpn_2x_coco.py` 的批次大小是 `32 x 4 = 128`, 或者 `scnet_x101_64x4d_fpn_8x1_20e_coco.py` 的批次大小是 `8 x 1 = 8`。
```

Comment:
`faster_rcnn_x101_32x4d_fpn_2x_coco.py` is wrong as an example and inconsistent with `docs/en/1_exist_data_model.md`.

Reply:
Done